packages/app/src/clickhouse.ts

```diff
@@ -89,7 +93,25 @@ export function useSessions(
       if (!traceSource || !sessionSource) {
         return [];
       }
-      // TODO: we
+
+      let whereClauseSQL = `1=1`;
```

Comment:
no need for fix, just a note in the future: I know we do this pattern a lot in v1, but it'd be nice in v2 if we started cleaning that up and just conditionally rendering the SQL statement since we aren't using the same SQL formatter as before.